cmd/soroban-rpc/internal/db/migration.go

```diff
@@ -27,10 +27,10 @@ func (mlr *LedgerSeqRange) IsLedgerIncluded(ledgerSeq uint32) bool {
 }
 
 func (mlr *LedgerSeqRange) Merge(other *LedgerSeqRange) *LedgerSeqRange {
```

Comment:
I know this is a pre-existing issue but I don't think `other` should be a pointer because that gives the impression that `Merge()` will modify `other` but we don't actually want to mutate `other` in `Merge()`

Reply:
furthermore, I think `Merge()` should return a `LedgerSeqRange` instead of a pointer. all of the LedgerSeqRange functions can have value receivers except for `MergeInPlace()` which modifies the ledger range struct. But I think we can get rid of `MergeInPlace()` (I'll elaborate on that in a separate comment)

Reply:
Yeah I definitely agree that these should be value receivers. Fixed in [a5bfc3b](https://github.com/stellar/soroban-rpc/pull/282/commits/a5bfc3b4d4270396a036850daea6226d421320ec)!